enzyme/tests/App.test.tsx

```diff
@@ -119,4 +120,10 @@ describe('App', () => {
         expect(component.state().childState.activeSmartContract).toBe(blueContract);
     });
 
+    it('attempts to post a message to vscode', async () => {
+        const postToVSCodeStub: sinon.SinonStub = mySandBox.stub(Utils, 'postToVSCode').resolves();
+        const component: any = mount(<App/>);
+        component.instance().postMessageHandler({});
+        postToVSCodeStub.should.have.been.called;
```

Comment:
Could you make this as strict as possible - `postToVSCodeStub.should.have.been.calledOnceWithExactly({});`